### ISSUE TITLE
Path to local assets?

### ISSUE THREAD

**Issue by torarnv** [vudash/vudash#73]
Some of the build in widgets take sounds, also by path. What is this patch if I want to keep it in the dashboard repository? Putting it at the root along with widgets and dashboards does not work, localhost:3300/foo.bar just gives me the index.

I'm trying to create a widget that just shows an image as configured by the option,  ` "url": "/assets/foo.png"`

**Comment by antony** [comment]
Hi @torarnv 

Currently there is no support for this, at least not in the existing widgets. You could create a custom widget which shows the image, however.

I think it might be worth adding a filesystem data source so that widgets such as the value widget can show images as well as text.

**Comment by torarnv** [comment]
Could the webserver you spin up serve out of the `public` folder of the user's project?

**Comment by antony** [comment]
@torarnv yeah - in fact it already does for internal assets:

https://github.com/vudash/vudash/blob/master/packages/core/src/plugins/static/static.js

It'd be a matter of adding a second handler in here which serves from a directory based on process.cwd()

**Comment by torarnv** [comment]
Right, that's what I saw. I'll play around with what you suggested! Thanks!
